source/components/ui/msm-modal/msm-modal.service.js

```diff
@@ -175,7 +175,7 @@
      *     The label of the modal's cancel button.
      */
     function select(title, text, options, size, closeTitle, dismissTitle) {
-      return open({ options: options }, function ($modalInstance, options) {
+      return open({ values: options.values, options: options }, function ($modalInstance, values, options) {
```

Comment:
Only direct children of the parameter object are resolved correctly. Thus, this should be `open({ values: options.values, selected: options.selected }, ...` not `open({ values: options.values, options: options }, ...`